### DISCUSSION TITLE
Is using `<Link>` always a better choice than using `<a>` only?

### DISCUSSION THREAD

**Discussion by mrcaidev** [vercel/next.js#37183] [Help] [answered]
I've been thinking about this question since I started to build my own website...

There are times when I **don't need any of the extra facilities** offered by `<Link>`, like "prefetch" or "as", but I think maybe Next.js has greatly optimized `<Link>` so that it will be a better choice to replace `<a>` with `<Link>` in most cases.

For example, here are two scenarios I encountered when I build my website.

## Scenario 1: Add an external link to GitHub

I don't need prefetch (as I think it will be somewhat expensive), so can I just use this?

```html
<a href="https://github.com" target="_blank" rel="noopener noreferrer">GitHub</a>
```

## Scenario 2: Jump to a fragment on current page

Like linking to `/#about` in navigation bar. And personally I don't see any point in using `<Link>`.

```html
<a href="/#about">About</a>
```

---

So can I just use `<a>` instead of `<Link>` in these scenarios? If not, why?

**Comment by icyJoseph** [comment, marked as answer]
I'd say yes, there's for sure cases where a plain `a`nchor tag gets the job done. For example, for an anchor tag that downloads a file, I wouldn't use a `Link` from `next/link`.

The real use of `Link` in any routing framework is to tell the router engine, that the user wishes the navigate, so that the router can deliver on the SPA promise, prevent page unload, and sort of, tell React, to render new/update UI.

In scenario 2, I'd say, if it is `/#about` I'd use the Link, because, perhaps I am not in the `/` route, when I click on the link. Which would make the Navigation bar component very context aware, and it'd trigger page unload if ever so unused incorrectly.
